generator/format/format.go

```diff
@@ -42,16 +42,33 @@ type StructFormatter interface {
 }
 
 type TypeFormatterRegistry struct {
-	KnownTypes map[types.Type]struct{}
 	Formatters []TypeFormatter
+
+	knownTypes      map[types.Type]struct{}
+	knownNamedTypes map[string]struct{}
+}
+
+func (t *TypeFormatterRegistry) AddKnownType(typ types.Type) {
```

Comment:
Did you start writing code in Polish? :D 
```suggestion
func (t *TypeFormatterRegistry) AddKnownType(type types.Type) {
```
<img width="1462" alt="image" src="https://github.com/user-attachments/assets/472cac56-f332-46d3-9c14-ec7100f3bb8d">

Reply:
`type` is a reserved word in Go, so I had to polish the code